docs/user-guide/command-line-interface.md

```diff
@@ -28,6 +28,7 @@ Options:
   -f, --format String         Use a specific output format - default: stylish
   -v, --version               Outputs the version number
   --reset                     Set all default rules to off - default: false
+  --base-config Boolean | path::String  Enable/disable base config, could be passed a file path - default: true
```

Comment:
I'd prefer not to expose this on the command line and only expose it on `CLIEngine`.